skills_taxonomy_v2/pipeline/skills_extraction/skills_naming_utils.py

```diff
@@ -20,69 +22,134 @@
 from skills_taxonomy_v2.pipeline.sentence_classifier.sentence_classifier import (
     BertVectorizer,
 )
+from skills_taxonomy_v2.getters.s3_data import load_s3_data, save_to_s3
+from skills_taxonomy_v2 import BUCKET_NAME
+
+from pattern.text.en import singularize
+from collections import OrderedDict
+
+from nltk.corpus import stopwords
+
+nltk.download("stopwords")
 
 logger = logging.getLogger(__name__)
 
 nltk.download("wordnet")
 
 
+def get_new_skills_embeds(new_skills_embeds_path, bucket_name):
+    """
+    takes as input string path to new skills embeddings directory and bucket name.
+    outputs:
+        dictionary of new skills embeddings.
+    """
+
+    s3 = boto3.resource("s3")
+
+    sentence_embeds = {}
+    all_sentence_embeds = []
+
+    for i in tqdm(range(0, 8)):  # 8 files
+
+        new_sentences_dict = load_s3_data(
+            s3, bucket_name, new_skills_embeds_path + f"sentences_data_{i}.json"
+        )
+        all_sentence_embeds.append(new_sentences_dict)
+
+    # https://stackoverflow.com/questions/57340332/how-do-you-combine-lists-of-multiple-dictionaries-in-python
+    for k in all_sentence_embeds[0].keys():
+        sentence_embeds[k] = sum(
+            [skills_dict[k] for skills_dict in all_sentence_embeds], []
+        )
+
```

Comment:
You may be able to use a default dict? Something like:
```
from collections import defaultdict
sentence_embeds = defaultdict(list)
for i in tqdm(range(0, 8)):  # 8 files
    new_sentences_dict = load_s3_data(
            s3, bucket_name, new_skills_embeds_path + f"sentences_data_{i}.json"
        )
    for k, v in new_sentences_dict.items():
        sentence_embeds[k].append(v)

```
Also to note instead of hardcoding the `range(0,8)` bit, I have done it this way:
```
reduced_embeddings_paths = get_s3_data_paths(
            s3,
            bucket_name,
            new_skills_embeds_path,
            file_types=["*sentences_data_*.json"]
            )

sentences_data = pd.DataFrame()
for reduced_embeddings_path in tqdm(reduced_embeddings_paths):
    sentences_data_i = load_s3_data(
                s3, bucket_name,
                new_skills_embeds_path
            )
    ...
    

```